.github/workflows/ci-unix-static-sanitized-memory.yml

```diff
@@ -0,0 +1,119 @@
+
+      - name: Build cxx and cxxabi
+        run: |
+          # clone LLVM
+          git clone --depth=1 --branch llvmorg-19.1.6  https://github.com/llvm/llvm-project
+          # configure cmake
+          cmake -G Ninja -S llvm-project/runtimes -B llvm-project/msan_out \
+                  -DCMAKE_BUILD_TYPE=Release \
+                  -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi;libunwind" \
+                  -DCMAKE_C_COMPILER=clang \
+                  -DCMAKE_CXX_COMPILER=clang++ \
+                  -DLLVM_USE_SANITIZER=MemoryWithOrigins \
+                  -DLIBCXXABI_USE_LLVM_UNWINDER=OFF
+          # build the libraries
+          cmake --build llvm-project/msan_out -- cxx cxxabi unwind
+      - name: Set FLAGS for memory sanitizer
+        run: |
+          echo "CI_CFLAGS=-fsanitize=memory -L${{ github.workspace }}/llvm-project/msan_out/lib -I${{ github.workspace }}/llvm-project/msan_out/include"  >> $GITHUB_ENV
+          echo "CI_CXXFLAGS=-fsanitize=memory -stdlib=libc++ -L${{ github.workspace }}/llvm-project/msan_out/lib -I${{ github.workspace }}/llvm-project/msan_out/include -I${{ github.workspace }}/llvm-project/msan_out/include/c++/v1"  >> $GITHUB_ENV
+          echo "CI_LDFLAGS=-fsanitize=memory -L${{ github.workspace }}/llvm-project/msan_out/lib -lc++abi"  >> $GITHUB_ENV
+          echo "CI_LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${{ github.workspace }}/llvm-project/msan_out/lib" >> $GITHUB_ENV
+
+      - name: Build jpeg
+        if: ${{ steps.cache-hit.outputs.hit == 'false' }}
+        working-directory: ./ext
+        run: |
+          sed -i -e 's/cmake -S \(.*\)/cmake -S \1 -DWITH_SIMD=OFF/g' libjpeg.cmd
+          ./libjpeg.cmd
+        env:
+          CFLAGS: ${{ env.CI_CFLAGS }}
+          CXXFLAGS: ${{ env.CI_CXXFLAGS }}
+          LDFLAGS: ${{ env.CI_LDFLAGS }}
+          LD_LIBRARY_PATH: ${{ env.CI_LD_LIBRARY_PATH }}
+
+      - name: Build aom
+        if: ${{ steps.cache-hit.outputs.hit == 'false' }}
+        working-directory: ./ext
+        run: |
+          sed -i -e 's/cmake -G Ninja \(.*\) \.\./cmake -G Ninja \1 -DAOM_TARGET_CPU=generic -DSANITIZE=memory ../g' aom.cmd
+          ./aom.cmd
+        env:
+          CFLAGS: ${{ env.CI_CFLAGS }}
+          CXXFLAGS: ${{ env.CI_CXXFLAGS }}
+          LDFLAGS: ${{ env.CI_LDFLAGS }}
+          LD_LIBRARY_PATH: ${{ env.CI_LD_LIBRARY_PATH }}
+
+      - name: Build dav1d
+        if: ${{ steps.cache-hit.outputs.hit == 'false' }}
+        working-directory: ./ext
+        run: |
+          sed -i -e 's/meson setup \(.*\) \.\./meson setup \1 -Db_sanitize=memory -Db_lundef=false -Denable_asm=false ../g' dav1d.cmd
+          ./dav1d.cmd
+        env:
+          CFLAGS: ${{ env.CI_CFLAGS }}
+          CXXFLAGS: ${{ env.CI_CXXFLAGS }}
+          LIBRARY_PATH: ${{ env.CI_LD_LIBRARY_PATH }}
+
+      - name: Prepare libavif (cmake)
+        run: >
+          cmake -S . -B build -G Ninja
+          -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=OFF
```

Comment:
Optional: Look into using `-DCMAKE_BUILD_TYPE=RelWithDebInfo`. Can do this in a follow-up pull request for all the sanitizers.

Reply:
Good catch, that was a leftover to debug some errors that popped at some point. Done